package/yast2-network.changes

```diff
@@ -1,3 +1,11 @@
+-------------------------------------------------------------------
+Mon Nov 23 08:03:17 UTC 2020 - Knut Anderssen <kanderssen@suse.com>
+
+- Do not show a warn message when modifying a bonding configuration
+  and all the slaves are already configured with BOOTPROTO='none'
+  (bsc#1178950)
+- 4.2.85
```

Comment:
Version number looks wrong (it should be ' 4.3.29`).

Reply:
Yep, I adapted the spec but do not touched the changes, as apparently I removed a line when I suggested the changes I will merge it again locally and will force the push.